parquet_fe_prototype/__init__.py

```diff
@@ -128,6 +128,30 @@ def create_app():
 
     datapackage, index = __build_search_index(app)
 
+    def sort_resources_by_name(resource):
```

Comment:
Kind of a janky way to sort a bunch of tables - pulled up into the "init" part of the function so we don't re-sort every time someone hits the site.